app/lib/browserConfig.js

```diff
@@ -21,6 +21,7 @@ requirejs.config({
       , 'socketio': '../socket.io/socket.io'
       , 'EventEmitter': 'bower_components/event-emitter.js/dist/event-emitter'
       , 'jszip': 'bower_components/jszip/dist/jszip'
+      , 'filesaver': 'bower_components/file-saver.js/FileSaver'
```

Comment:
It would be better to name the module here `saveAs` so it's more obvious what you require. This FileSaver module exports just one function called `saveAs`.